### DISCUSSION TITLE
Why are there GitHub releases for every npm package in this repository?

### DISCUSSION THREAD

**Discussion by Nautigsam** [verdaccio/verdaccio#2164] [Q&A] [answered]
I use GH notifications to know when a new version of verdaccio is out. This morning I got spammed by GH because a new pre-release was published for every npm package "@verdaccio/...". I turned the notifications off after the 35th email.

Is this a mistake or is this intended?

**Comment by eren-amundsen** [comment]
Me too

**Comment by DanielRuf** [comment, marked as answer]
Do you mean the tags / releases at https://github.com/verdaccio/verdaccio/releases?

If so, yes this is intended. GitHub creates these entries for tags and they are automatically published.

**Reply by Nautigsam** [reply]
That's a shame, it makes it impossible to enable notifications for new releases.

I still wonder what is the point in publishing these releases since they are included in verdaccio anyway?

**Reply by sergiohgz** [reply]
Our point of view is allow users to test things done to find possible issues ASAP.

In the other hand, I would suggest you to request Github to allow filtering in this kind of subscriptions. For example, if you could filter out prereleases notifications, you would have what you really need.

Sorry for the inconvenience

**Reply by Nautigsam** [reply]
I guess I can filter-out email notifications for pre-releases that's ok. Thanks for the response.

**Comment by juanpicado** [comment]
@Nautigsam sorry for that, I just merged `changesets` on master and I did not realized that creates a GH releases peer package, I'll find the way to disable that, I agree too spam-my, not cool.
